src/lib/components/workflow-actions.svelte

```diff
@@ -206,23 +206,22 @@
             workflowType: workflow.name,
           }),
         )}
+      disabled={workflowCreateDisabled($page)}
```

Comment:
Wondering if we want some tooltip text here for disabled now that we're not just hiding the menu item.

Reply:
Probably? We're going to refactor these to a new UI instead of the split button, so I think this is okay for now.

Reply:
We have a bunch of spots where we just disable if read-only so I'm okay with this